packages/sdk/test/unit/node/core/onBehalfOfUserCredential.spec.ts

```diff
@@ -65,6 +65,8 @@ describe("OnBehalfOfUserCredential Tests - Node", () => {
     mockedEnvRestore = mockedEnv({
       M365_CLIENT_ID: clientId,
       M365_CLIENT_SECRET: clientSecret,
+      // M365_CERTIFICATE_PATH:
```

Comment:
do we need these two lines? if not clean up?

Reply:
Will create a new PR